pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -682,30 +675,31 @@ func (cj *changefeedJob) waitForCompletion() {
 type opt func(ct *cdcTester)
 
 // withNumSinkNodes sets the number of nodes to use for sink nodes. Only Kafka has been tested currently.
+// N.B. this allocates a workload + sink only node, without this workload and sink will share a node.
```

Comment:
Nit: "...without it, workload and sink will be on the same node."

Reply:
Done